components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.html

```diff
@@ -5,22 +5,17 @@
         <chef-heading>Chef Servers</chef-heading>
         <chef-subheading>Manage Chef Servers with Chef Automate.</chef-subheading>
       </chef-page-header>
-      <app-create-chef-server-modal
-        [visible]="createModalVisible"
-        [creating]="creatingChefServer"
-        [createForm]="createChefServerForm"
-        (close)="closeCreateModal()"
-        [conflictErrorEvent]="conflictErrorEvent"
+      <app-create-chef-server-modal [visible]="createModalVisible" [creating]="creatingChefServer"
+        [createForm]="createChefServerForm" (close)="closeCreateModal()" [conflictErrorEvent]="conflictErrorEvent"
         (createClicked)="createChefServer()">
       </app-create-chef-server-modal>
-      <app-delete-object-modal
-        [visible]="deleteModalVisible"
-        objectNoun="server"
-        [objectName]="serverToDelete?.name"
-        (close)="closeDeleteModal()"
-        (deleteClicked)="deleteServer()"
-        objectAction="Delete">
+      <app-delete-object-modal [visible]="deleteModalVisible" objectNoun="server" [objectName]="serverToDelete?.name"
+        (close)="closeDeleteModal()" (deleteClicked)="deleteServer()" objectAction="Delete">
       </app-delete-object-modal>
+      <app-message-modal [title]="'Could Not Delete Server'" [visible]="messageModalVisible"
+        (close)="closeMessageModal()">
+        Before you can delete this server, delete all orgnisations attached to this server.
+      </app-message-modal>
       <div class="page-body">
```

Comment:
@chef/ui-team is there consensus on listing many attributes on their own line or only wrapping them at a certain line length?

```
<app-create-chef-server-modal                                                                 |
  [visible]="createModalVisible"                                                              |
  [creating]="creatingChefServer"                                                             |
  (close)="closeCreateModal()"                                                                |
   ...etc...>                                                                                 |
</app-create-chef-server-modal>                                                               |
```

vs

```
<app-create-chef-server-modal [visible]="createModalVisible" [creating]="creatingChefServer"  |
   (close)="closeCreateModal()" ...etc...>                                                    |
</app-create-chef-server-modal>                                                               |
```

I've usually preferred to keep longer lists of attributes/prop bindings on their own line but don't mind either really. By default vscode wraps them by line whenever you copy/paste html blocks

Reply:
I used to be indifferent, but lately I am leaning towards one-per-line; it greatly improves readability.

Reply:
changed as one-per-line :+1:

Reply:
@msorens yea same. Better readability in file and in git diffs. Thx @sachin-msys :+1: